lib/ukfallocbuddy/Config.uk

```diff
@@ -26,4 +26,18 @@ config LIBUKFALLOCBUDDY_STATS
 	bool "Collect frame allocation statistics"
 	default n
 
+config LIBUKFALLOCBUDDY_TEST
+	bool "Enable tests"
+	default n
+	select LIBUKTEST
+
+if LIBUKFALLOCBUDDY_TEST
+
+config LIBUKFALLOCBUDDY_PUBLIC_FUNCTIONS
```

Comment:
What exactly is the purpose of the `LIBUKFALLOCBUDDY_PUBLIC_FUNCTIONS`?
From what I can see it does not seem to do anything, did I miss something?

Reply:
Initially, I proposed to @marcrittinghaus  there should be 2 sources that contain tests for `ukfallocbuddy` library. One should contain tests with the functions that are in the `fallocbuddy.h` file and one contains tests with all static functions that are present in the `ukfallocbuddy/fallocbuddy.c`. After looking at it better I saw no need add one more. I've erased it now.